VSConfigFinder/Program.cs

```diff
@@ -38,17 +39,12 @@ private static void Run(CommandLineOptions options)
                 options.ConfigOutputPath ??= Directory.GetCurrentDirectory();
```

Comment:
Now that we have all the parts, should we pull out this if check then assign part to something like `ResolveOptions(CommandLineOptions options)`?